partner_company_group/models/account_invoice.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    company_group_id = fields.Many2one(
+        related="partner_id.company_group_id", store=True
```

Comment:
```suggestion
        related="partner_id.company_group_id", store=True, readonly=True,
```
apply the same in other files

by default in v10, related and compute were readonly=False.